rpp-charter.md

```diff
@@ -3,7 +3,8 @@
 The Extensible Provisioning Protocol (EPP) was standardized ([STD69](https://datatracker.ietf.org/doc/std69/)) 
 in 2009 to address the needs of domain name management between domain name registries and registrars.
 Though EPP is still serving the domain name industry well, the progress in available development, integration and operational
-patterns, tools and technologies create a desire to have a provisioning protocol using the REST architectural style and the JSON data-interchange format.
+patterns, tools and technologies create a desire to have a provisioning protocol using the [REST](https://ics.uci.edu/~fielding/pubs/dissertation/rest_arch_style.htm) 
```

Comment:
I think this is the right reference: http://www.ics.uci.edu/~fielding/pubs/dissertation/top.htm
At least this is how it's linked on the author's own publication page: https://roy.gbiv.com/vita.html

Reply:
That's the reference for the full paper. The link I provided goes specifically to Section 5, the REST architectural style.

Reply:
I know, I was just trying to follow the general pattern of references to the full publication, same as in other IETF documents, not deep link to the specific chapter.
This was just a remark, I can live with deep link as well.